test/unit/logic/transaction.js

```diff
@@ -679,6 +679,17 @@ describe('transaction', function () {
 			});
 		});
 
+		it('should return error on timestamp out of int32 range', function (done) {
+			var trs = _.cloneDeep(validTransaction);
+			trs.timestamp = -2147483648 - 1;
+			delete trs.signature;
+			trs.signature = transaction.sign(senderKeypair, trs);
+			transaction.verify(trs, validSender, null, true, function (err) {
+				expect(err).to.include('Invalid transaction timestamp');
```

Comment:
Please expect exact message, part of it it's not reliable.

Reply:
done